assets/js/amp-story-editor-blocks.js

```diff
@@ -131,14 +219,26 @@ var ampStoryEditorBlocks = ( function() { // eslint-disable-line no-unused-vars
 	 * @return {Object} Settings.
 	 */
 	component.addAMPAttributes = function addAMPAttributes( settings, name ) {
-		// Add the "thirds" template position option.
+		// Add the "thirds" template position option and animation settings.
 		if ( -1 !== component.data.allowedBlocks.indexOf( name ) ) {
 			if ( ! settings.attributes ) {
 				settings.attributes = {};
 			}
 			settings.attributes.ampStoryPosition = {
 				type: 'string'
 			};
+			// @todo We could map all the blocks to their tag and use attribute as source instead.
```

Comment:
Or maybe it's rather because we don't know the element name to select?

Reply:
Changed these settings to `source: attribute` by using `parseInt()` within 4923569. Exception is for `core/list` since it could be either `ul` or `ol`.